README.md

```diff
@@ -45,90 +45,77 @@ As the first step, you can build a Ballerina service that gives the current time
 
 ```ballerina
 import ballerina/http;
-import ballerina/io;
 import ballerina/time;
 
-endpoint http:Listener listener {
-    port: 9095
-};
+listener http:Listener timeEP = new(9095);
 
 @http:ServiceConfig { basePath:"/localtime" }
-service<http:Service> time bind listener {
+service time on timeEP {
     @http:ResourceConfig {
         path: "/",
         methods: ["GET"]
     }
-    getTime (endpoint caller, http:Request request) {
+    resource function getTime (http:Caller caller, http:Request request) {
         http:Response response = new;
         time:Time currentTime = time:currentTime();
         string customTimeString = currentTime.format("yyyy-MM-dd'T'HH:mm:ss");
-        json timeJ = { currentTime : customTimeString };
+
+        json timeJ = {currentTime : customTimeString };
         response.setJsonPayload(timeJ);
         _ = caller -> respond(response);
     }
 }
+
 ```
 
 Now you can add the Kubernetes annotations that are required to generate the Kubernetes deployment artifacts. 
 
 ```ballerina
 import ballerina/http;
-import ballerina/io;
 import ballerina/time;
 import ballerinax/kubernetes;
 
-@kubernetes:Ingress {
-    name: "ballerina-time-service",
-    path: "/localtime",
-    ingressClass: "istio"
-}
+@kubernetes:IstioGateway {}
+@kubernetes:IstioVirtualService {}
 @kubernetes:Service {
-    serviceType: "NodePort",
     name: "ballerina-time-service"
 }
-endpoint http:Listener listener {
-    port: 9095
-};
+listener http:Listener timeEP = new(9095);
 
 @kubernetes:Deployment {
     image: "ballerina-time-service",
     name: "ballerina-time-service",
     singleYAML: true
 }
-@http:ServiceConfig {basePath:"/localtime"}
-service<http:Service> time bind listener {
+@http:ServiceConfig { basePath:"/localtime" }
+service time on timeEP {
     @http:ResourceConfig {
-        path: "/",  
+        path: "/",
         methods: ["GET"]
     }
-    getTime (endpoint caller, http:Request request) {
+    resource function getTime (http:Caller caller, http:Request request) {
         http:Response response = new;
         time:Time currentTime = time:currentTime();
         string customTimeString = currentTime.format("yyyy-MM-dd'T'HH:mm:ss");
+
         json timeJ = {currentTime : customTimeString };
         response.setJsonPayload(timeJ);
```

Comment:
lets pass the payload directly to respond method

Reply:
Fixed with https://github.com/ballerina-guides/ballerina-with-istio/pull/23/commits/76ef7c0249bc6ea5a5d1b1919082479cc3de49a6